app/javascript/vue/shared/content/attachments/thumbnail.vue

```diff
@@ -246,6 +247,9 @@ export default {
   methods: {
     openOVEditor(url) {
       window.showIFrameModal(url);
+    },
+    openScinoteEditor() {
+      $(this.$refs.imageEditButton).trigger('click');
```

Comment:
We want to avoid using jQuery unless absolutely necessary. Just use native javascript: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click